docs/cloud-adoption/governance/governance-multiple-teams.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Governance design guide: new development in Azure for multiple teams"
+title: "Enterprise Cloud Adoption: Governance design in Azure for multiple teams"
 description: Guidance for configuring Azure governance controls for multiple teams, multiple workloads, and multiple environments
-author: petertay
+author: petertaylor9999
 ---
 
-# Governance design guide: new development in Azure for multiple teams
+# Enterprise Cloud Adoption: Governance design for multiple teams
 
-The goal of this guidance is to help you learn the process for designing a resource governance model in Azure.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support multiple teams, multiple workloads, and multiple environments.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 Our requirements are:
```

Comment:
Using "our" here feels odd to me. Could say "The requirements" and "The individual in *your* organization"?

Reply:
Yep, this is from when it was more progressive guidance and conversational in tone. This is fixed.